nbcparser/go.mod

```diff
@@ -0,0 +1,31 @@
+module github.com/Azure/agentbaker/nbcparser
```

Comment:
can we ensure we have Dependabot configured for AgentBaker to have those dips handled with automatic CVE updated.

Reply:
dips? not sure i follow, can you elaborate?

Reply:
deps (dependancies)

Reply:
Just double checked with Cameron that we already have a dependabot configured for AB